.github/actions/verify-token/src/predicate1.ts

```diff
@@ -14,32 +14,18 @@ limitations under the License.
 import * as github from "@actions/github";
 import * as fs from "fs";
 
-import {
-  githubObj,
-  rawTokenInterface,
-  ArtifactReference,
-  SLSAv1Predicate,
-} from "./types";
+import { rawTokenInterface } from "./types";
+import { createURI, getWorkflowPath } from "./utils";
+import { SLSAPredicate, ArtifactReference } from "./slsatypes1";
 
-const DELEGATOR_BUILD_TYPE =
-  "https://github.com/slsa-framework/slsa-github-generator/delegator-generic@v0";
-
-// getWorkflowPath returns the workflow's path from the workflow_ref.
-export function getWorkflowPath(obj: githubObj): string {
-  // GITHUB_WORKFLOW_REF contains the repository name in the path. We will trim
-  // it out.
-  // e.g. 'octocat/hello-world/.github/workflows/my-workflow.yml@refs/heads/my_branch'
-  // Strip off the repo name and git ref from the workflow path.
-  return obj.workflow_ref
-    .substring(`${obj.repository}/`.length)
-    .split("@", 1)[0];
-}
+const DELEGATOR_BUILD_TYPE_V1 =
+  "https://github.com/slsa-framework/slsa-github-generator/delegator-generic@v1";
```

Comment:
the other file versions the delegator build type at v0, why differ?

Reply:
I thought a bit about this and went this way because it was different SLSA versions, but I guess it doesn't really matter much. I'll just make them both v0.